.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
+.vscode/
```

Comment:
What do we need to specifically ignore from .vscode folder?
Sometimes it's nice to share project settings for all

Reply:
Usually they are vscode preferences, doesnt it? Might be different from person to person.
If not, there's no problem for me to commit it